app/events/page.tsx

```diff
@@ -0,0 +1,12 @@
+// app/events/page.tsx
+import { client } from 'tina/__generated__/client';
+import EventsClient from './EventsClient';
+
+export default async function EventsPage() {
+  const vars = {};
+
+  const res = await client.queries.eventsConnection(vars);
```

Comment:
The `getStaticProps` functions which was used in old Pages Router is handled directly in `pages.tsx` and passed to client component as props when migrating to App Router.
Please read this documentation for more info - https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#static-site-generation-getstaticprops

For example:
```
const getEvents = async () => {
  const res = await client.queries.eventsConnection(vars);
  return {
    props: {
      data: res.data
      data: res.query
    }
  }
}
```

And then when calling the component from `page.tsx`:
```
export default async function EventsPage() {
  const { props } = await getEvents();

  return <EventsClient props={props} />;
}
```

Reply:
Done